relay-server/src/services/processor/span/processing.rs

```diff
@@ -528,6 +509,32 @@ fn normalize(
     Ok(())
 }
 
+fn populate_ua_fields(
+    span: &mut Span,
+    request_user_agent: Option<&str>,
+    mut client_hints: ClientHints<&str>,
+) {
+    let data = span.data.value_mut().get_or_insert_with(SpanData::default);
+
+    let user_agent = data.user_agent_original.value_mut();
```

Comment:
just to confirm for a Relay/Rust noob: We convert `user_agent.original` to `user_agent_original` somewhere, prior to accessing it here, correct?